src/core/common/constants.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export type MessageTypes = [
+  "generalError",
+  "parsingDataError",
+  "constraintSolvingError",
+  "linkGuideCreatingError",
+  "invalidLinksData",
+  "noID",
+  "noSourceOrTargetID"
+];
+
+export const messageTypes: MessageTypes = [
```

Comment:
I think it makes sense here to use an enum like: https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-compile-time

```ts
export enum MessageType {
  GeneralError,
  ParsingDataError,
  ....
}

export const MessageTypeKeys = keyof typeof MessageType;
```

Reply:
I added `export const messageTypes = Object.values(MessageType);`